package.json

```diff
@@ -6,7 +6,7 @@
   "license": "AGPL-3.0-only",
   "repository": "https://github.com/mergeability/mergeable.git",
   "scripts": {
-    "dev": "LOG_LEVEL=trace NODE_ENV=development nodemon --exec 'npm start'",
+    "dev": "LOG_LEVEL=info NODE_ENV=development nodemon --exec 'npm start'",
```

Comment:
Is this intentional?

Reply:
actually, i think we should remove the LOG_LEVEL, it hasn't been helpful at all and we can individually set it if we need it. Done